### ISSUE TITLE
Manifest

### ISSUE THREAD

**Issue by Poussinou** [freshollie/monkeyboard-radio-android#16]
Hi, can you please fix the problems in your repo we are talking about on this [page](https://gitlab.com/IzzyOnDroid/repo/issues/3) ?

By the way, are you ok with the inclusion of your app in the [IzzyOnDroid Fdroid compatible repo](https://apt.izzysoft.de/fdroid/) (more info [here](https://apt.izzysoft.de/fdroid/index/info)) ? This will allow users to have automatic updates of your app trough F-Droid if they have enabled this specific repo. The only thing you will have to do is to tag the releases properly, like specified in this discussion on Gitlab.

**Comment by freshollie** [comment]
Ok, so all I need to do is add the dash in the versioncode like there is in the git tag?

**Comment by Poussinou** [comment]
Something like that yes 👍 

@IzzySoft ?

**Comment by IzzySoft** [comment]
@freshollie Good practice (and used by the majority of Android projects at Github) is to have the tag name match the `versionName`, only prefixed by a `v`. A good naming scheme (matching that to your latest release) would thus be a `versionName` of e.g. `2.1.3.beta` with a tag of `v2.1.3.beta` (variations exist like `2.1.3-beta` or `2.1.3_beta`). That's also what F-Droid itself recommends. To bring it in my repo, it would also suffice to simply adjust the `versionName` to `beta-2.1.3` or `beta-v2.1.3` (my updater should be able to match that). But as in my case the `versionName` becomes part of the file name, it must not contain spaces (causes trouble with some clients).

TL;DR: if the both match, and there are no spaces, I could take it in.

If then (when a "matching release" is available) one of you updates the ticket referenced in the initial post, I'd know I can start setting things up :wink:

**Comment by freshollie** [comment]
@IzzySoft Thanks very much for the info. The `versionName` has been updated in [this branch](https://github.com/freshollie/monkeyboard-radio-android/tree/polling-change), however next release I will changing the version naming scheme to `2.1.x-beta` to conform to convention.

Cool repo. You are correct that this app is not used by many however it is great that it is possible to be integrated into your repository. After the next release I will update the readme to include a link on how to use your repo in F-Droid to provide automatic software updates.

I'm currently living in Laim ;)

**Comment by IzzySoft** [comment]
Actually, I've came upon your app already in the past on my scans at Github – but due to the requirement of specific hardware *and* no `.apk` being present at the time (IIRC) I've skipped it. But as @Poussinou brought it up again, I thought I'd give it a run :wink:

Thanks for your compliments on my repo, @freshollie – glad to see you like the idea of your app being included there! And Laim I'll pass by next Tuesday again on my way to Pasing (have to give another Android course at the VHS there – beginners courses, so nothing you'd feel interesting to attend).

**Comment by freshollie** [comment]
You are not mistaken. I only started building release APKs in November. Small world though haha. I will comment when this is fixed.

**Comment by freshollie** [comment]
Ok @IzzySoft this should be fixed with [the latest release](https://github.com/freshollie/monkeyboard-radio-android/releases/tag/v2.1.4-beta). Let me know if it's all ok.

**Comment by IzzySoft** [comment]
Thanks Oli! Let me answer your link [with another one](https://apt.izzysoft.de/fdroid/index/apk/com.freshollie.monkeyboard.keystoneradio) – and be welcome to mention that as "installation option with automatic updates" :wink:
